css/properties/backdrop-filter.json

```diff
@@ -57,7 +57,7 @@
               "version_added": "9"
             },
             "safari_ios": {
-              "version_added": null
+              "version_added": false
```

Comment:
Alright, sounds like taking prefixes into account is necessary to avoid adding false values which would be pedantically correct but not as useful as listing support for the prefixed property.

These results were collected before I was generating tests for prefixed forms based on `prefix` properties and `alternative_name`, so what I'll do is push a commit to revert the changes in this PR and send a PR with manual changes for these bits. I'll also file an issue about handling prefixed properties when making updates, and try it out on the state of the repo without this PR merged.

(In other words, I'd like to revert the wrong changes and get the good ones merged soon.)

Reply:
Sounds good to me. Thanks for going the useful path instead of the pedantically correct one :)